react/components/LayoutContainer.tsx

```diff
@@ -61,7 +66,7 @@ class Container extends Component<ContainerProps, ContainerState> {
 
     const returnValue: JSX.Element[] = elements.slice(0, this.state.elementsToRender).map((element: Element) => {
       return (
-        <Container key={element.toString()} elements={element} isRow={!isRow} {...props} aboveTheFold={undefined} >
+        <Container key={element.toString()} elements={element} isRow={!isRow} {...props} aboveTheFold={undefined} preview={undefined}>
```

Comment:
You can simply extract `aboveTheFold` and `preview` props in the destructuring of line 53 instead of explicitly passing undefined.